nerdlandbot/commands/GuildData.py

```diff
@@ -159,6 +161,20 @@ def user_is_admin(self, user_to_check: Member):
             or user_to_check.id in self.bot_admins
         )
 
+    def user_is_botslet(self, user_to_check: Member):
```

Comment:
funny but not sure if this is going to be a good name for others to read and understand from the name of the function what it does, would recommend changing to something like `is_admin_moderator()`